scripts/build.sh

```diff
@@ -49,5 +49,6 @@ else
 fi
 
 # Build AWM Relayer, which is run as a standalone process
-echo "Building AWM Relayer Version: $(git describe --tags --abbrev=0) at $binary_path"
-go build -o "$binary_path" "main/"*.go
+last_git_tag=$(git describe --tags --abbrev=0)
```

Comment:
This will fail if no tag is found. I recommend setting it to the same default value hardcoded in the binary in that case.
```suggestion
last_git_tag=$(git describe --tags --abbrev=0 2>/dev/null) || last_git_tag="v0.0.0-dev"
```